cfgov/apache/conf.d/alias.conf

```diff
@@ -11,3 +11,4 @@ Alias /robots.txt ${STATIC_PATH}/robots.txt
 Alias /static/f/ ${STATIC_PATH}/nemo/_/f/
 # Preserves legacy URL compatibility, but this is not actually using Wordpress
 Alias /wp-content/themes/cfpb_nemo/_/  ${STATIC_PATH}/nemo/_/
+Alias /static/ ${STATIC_PATH}/
```

Comment:
The comment in line 1 (`# These paths are served by WSGI as configured in wsgi.conf.`) is no longer accurate, since these aren't served by WSGI, but are served by Apache. Can it be removed?

Also, if we're aliasing `/static/` to `${STATIC_PATH)/`, then can/should we remove most of the explicit list of files above (e.g. `Alias /code.json ${STATIC_PATH}/code.json`, since they're now redundant? The only ones that aren't redundant are `/static/f` to `${STATIC_PATH}/nemo/_/f/` and the `wp-content` one.

Reply:
I'll certainly remove the comment, but for the second part I don't think they're actually redundant, because the url is, eg, `consumerfinance.gov/code.json` which needs to be mapped to `<cfgov_path>/static/code.json`. The only way they'd be redundant is if we changed all the urls to be in the form of  `consumerfinance.gov/static/code.json`, which I don't think we want to do.

Reply:
Ack, you're right. I totally misread that. Thanks for the correction.